lib/services/android-project-service.ts

```diff
@@ -504,6 +492,19 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 
 		return versionInManifest;
 	}
+
+	private async executeGradleCommand(projectRoot: string, gradleArgs: string[], childProcessOpts?: SpawnOptions, spawnFromEventOptions?: ISpawnFromEventOptions): Promise<ISpawnResult> {
+		const gradleBin = this.$hostInfo.isWindows ? "gradlew.bat" : "./gradlew";
```

Comment:
I would rename this const to "gradlew" as it points to the wrapper, not gradle itself.